pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -264,10 +269,8 @@ func (a Adapter) Push(ctx context.Context, parameters adapters.PushParameters, c
 	}
 	componentStatus.PostStartEventsDone = true
 
-	cmdKind := devfilev1.RunCommandGroupKind
 	cmdName := parameters.DevfileRunCmd
 	if parameters.Debug {
-		cmdKind = devfilev1.DebugCommandGroupKind
 		cmdName = parameters.DevfileDebugCmd
 	}
```

Comment:
Can we move `cmdName` along with `cmdKind` as well for a better readability?

Reply:
Done in [`d1a7381` (#6576)](https://github.com/redhat-developer/odo/pull/6576/commits/d1a738181381da83e959b8efa5bdc782f5030d99)